### PR TITLE
Apger - adding Wiz Collector.  adding RiskAndMorty Collector.

### PR DESCRIPTION
**Pull request by apger**
README.md files also included.

### CODE REVIEW

File: collectors/rest/wiz/wiz-auditlogentries.json
Comment (bdalpe) [inline collectors/rest/wiz/wiz-auditlogentries.json:1]: @apger can you preface the collector template files with `collector-`? 

e.g. `wiz-auditlogentries.json` becomes `collector-wiz-auditlogentries.json` 

Thanks!

**Review by bdalpe — CHANGES_REQUESTED** [review]

File: collectors/rest/wiz/wiz-auditlogentries.json
Comment (apger) [reply collectors/rest/wiz/wiz-auditlogentries.json:1]: Done.  Thanks.

**Comment by apger** [comment]
Prefaced the collector template files with collector- for the Wiz collector files

**Review by bdalpe — APPROVED** [review]
LGTM!

**Pull request merged**